SpatialGDK/Source/SpatialGDK/Public/Utils/InterestFactory.h

```diff
@@ -30,7 +30,10 @@
  * made by that server worker.
  */
 
+#if WITH_GAMEPLAY_DEBUGGER
```

Comment:
nit: no need to guard forward declarations technically since they don't themselves depend on having the gameplay debugger build available (but it also won't hurt, so feel free to ignore)